plugins/amazonq/shared/jetbrains-community/resources/software/aws/toolkits/resources/AmazonQBundle.properties

```diff
@@ -1,3 +1,11 @@
 action.q.hello.description=Hello description
+amazonqInlineChat.hint.edit = Edit
+amazonqInlineChat.popup.accept=Accept \u23CE
+amazonqInlineChat.popup.cancel=Cancel \u238B
+amazonqInlineChat.popup.confirm=Confirm \u23CE
```

Comment:
unclear if product only wants these for mac, or all OS

Reply:
ideally it should be different for windows we are tracking that as a P2